src/ParallelHashMap.h

```diff
@@ -244,7 +244,7 @@ void FixedHashMap<K,V>::insert(K key, V value)
   /* check to see if key already exists in map */
   if (contains(key))
     return;
-
+ 
```

Comment:
Hmm, I'm not sure where this whitespace is coming from in this file. Are these changes from @geogunow's PR #270?